### PR TITLE
pkp/pkp-lib#2135 Password reset process update

### PR DESCRIPTION
**Pull request by touhidurabir**
This PR aims to solve pkp/pkp-lib#2135

### CODE REVIEW

File: locale/ro_RO/emails.po [hunk 1]
```diff
@@ -159,7 +140,56 @@ msgstr ""
 "să consultați aici cuprinsul și apoi să vizitați site-ul nostru web pentru a "
 "citi articole de interes.<br />\n"
 "<br />\n"
+<<<<<<< HEAD
```
Comment (asmecher) [inline locale/ro_RO/emails.po:143]: @touhidurabir, here's a merge conflict remnant.

Comment (touhidurabir) [reply locale/ro_RO/emails.po]: Fixed .

File: locale/bs_BA/emails.po [hunk 2]
```diff
@@ -15,8 +15,781 @@ msgstr ""
 "%10<=4 && (n%100<10 || n%100>=20) ? 1 : 2;\n"
 "X-Generator: Weblate 3.9.1\n"
 
-msgid "emails.notification.subject"
-msgstr "Nova obavijest od {$siteTitle}"
+msgid "emails.passwordResetConfirm.subject"
```
Comment (asmecher) [inline locale/bs_BA/emails.po:18]: @touhidurabir, are all these changes intentional?

File: locale/fa_IR/emails.po [hunk 3]
```diff
@@ -10,6 +10,874 @@ msgstr ""
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
+"POT-Creation-Date: 2019-12-02T16:17:19-08:00\n"
+"PO-Revision-Date: 2019-12-02T16:17:19-08:00\n"
+"Language: \n"
+
+msgid "emails.passwordResetConfirm.subject"
```
Comment (asmecher) [inline locale/fa_IR/emails.po:17]: Likewise, not sure all these changes are intentional.

**Review by asmecher — CHANGES_REQUESTED** [review]
Found a few changes to locale files that look accidental to me.

**Review by asmecher — APPROVED** [review]

File: locale/bs_BA/emails.po (hunk 2 above)
Comment (touhidurabir) [reply locale/bs_BA/emails.po]: Fixed.

File: locale/fa_IR/emails.po (hunk 3 above)
Comment (touhidurabir) [reply locale/fa_IR/emails.po]: Fixed.

**Comment by asmecher** [comment]
Cherry-picked manually due to submodule conflict.

**Pull request closed without merging**